app/views/billables/_filters.html.haml

```diff
@@ -2,7 +2,7 @@
 = simple_form_for(@filters, url: billables_path, method: "get") do |f|
   = f.input :client_id, collection: @filters.clients, label: false, include_blank: t("entry_filters.clients")
   = f.input :project_id, collection: @filters.projects, label: false, include_blank: t("entry_filters.projects")
-  = f.input :from_date, as: :date, html5: true
-  = f.input :to_date, as: :date, html5: true
+  = f.input :from_date, input_html: { class: "datepicker", value: @filters.from_date ? @filters.from_date.strftime(I18n.t("date.formats.default")) : "" }
+  = f.input :to_date, input_html: { class: "datepicker", value: @filters.to_date ? @filters.to_date.strftime(I18n.t("date.formats.default")) : "" }
```

Comment:
Would be nice if we could clean this up a bit eventually, maybe make it a helper or move it into the filter, fine for now though ^^